preupg/xmlgen/xml_utils.py

```diff
@@ -209,16 +207,17 @@ def check_script_modification(self, key, k):
         self.update_values_list(self.rule, "{" + k + "}", key[k])
 
     def prepare_sections(self):
-        """The function prepares all tags needed for generation group.xml file."""
-        for main, self.keys in self.ini_files.items():
-            if main.endswith("group.ini"):
+        """The function prepares all tags needed for generation group.xml file.
+        """
+        for ini_filepath, ini_file_content in iter(self.ini_files.items()):
```

Comment:
You are right, that behaviour inside *for* loop is same and probably redundant in most cases, as the `self.ini_files.items()` returns list of items, which is naturaly iterated by *for* loop.